lib/note-editor/index.tsx

```diff
@@ -139,14 +140,15 @@ export class NoteEditor extends Component<Props> {
   };
 
   render() {
-    const { editMode, note, noteBucket, fontSize } = this.props;
+    const { editMode, isSmallScreen, note, noteBucket, fontSize } = this.props;
     const revision = this.props.revision || note;
     const tags = (revision && revision.data && revision.data.tags) || [];
     const isTrashed = !!(note && note.data.deleted);
 
     return (
       <div className="note-editor theme-color-bg theme-color-fg">
         <NoteDetail
+          isSmallScreen={isSmallScreen}
```

Comment:
I don't think this is needed due to https://github.com/Automattic/simplenote-electron/pull/2037/files#diff-52e457c4f342fbeecf796739f2492ccaR385